src/grafana_dashboards/minio-overview_rev13.json.tmpl

```diff
@@ -284,7 +284,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "topk(1, sum(minio_cluster_capacity_usable_free_bytes{job=\"$scrape_jobs\"}) by (instance))",
+          "expr": "topk(1, sum(minio_cluster_capacity_usable_free_bytes{}) by (instance))",
```

Comment:
this dashboard is not showing data for me 
![image](https://github.com/canonical/minio-operator/assets/49988746/f26fc632-adda-4cba-a012-4948a292509c)

Reply:
we possibly need to change it to [one of those metrics](https://min.io/docs/minio/kubernetes/upstream/operations/monitoring/metrics-and-alerts.html#id6)

Reply:
Changing to `minio_node_disk_free_bytes`

Reply:
Discussing with Noha, we decided to revert this and remove those disk metrics after all since in our case MinIO doesn't provide them.
Looking at this issue [comment](https://github.com/canonical/bundle-kubeflow/issues/855#issuecomment-2029573498), where metrics from each application at a certain time are exposed, `minio.txt` doesn't include those metrics. In general, we can expect applications to output more metrics than those there, but I guess in this case and since this is a generic metric, this won't happen.

Reply:
looks good now